docs/sources/write/shortcodes/index.md

```diff
@@ -154,6 +154,62 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo built-in `relref` shortcode.
+
+Use this code when content from one repository is published to more than one documentation set, because it lets you specify appropriate links for each doc set in one part of the file (usually at end of the file, like a footer) while using the link label in the body text.
+
+For example, a page in versioned Grafana documentation is also mounted in the Grafana Cloud documentation.
+The page in Grafana should link to the Grafana dashboards page but the page in Grafana Cloud should link to the Grafana Cloud dashboards page.
+
+Set the reference at the end of the document as follows:
+
+```markdown
+{{%/* docs/reference */%}}
+[dashboards]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/dashboards"
+[dashboards]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/dashboards"
+{{%/* /docs/reference */%}}
+```
+
+The content within the shortcode tags is as follows:
+
+- `label` - The label for the shortcode. This is the text that you'll use in the body text. In the example above, the label is `dashboards`. The label can be multiple words (for example, [dashboard docs]) and can include spaces.
+- `project path prefix` - Designates the destination project. In the example above, the path prefixes are `/docs/grafana/` for Grafana and `/docs/grafana-cloud/` for Cloud.
```

Comment:
@JStickler - you left the following question on the previous version of this PR:

> Are these paths in the `website` repo? If yes, it would be helpful to document that.

The Cloud project is in the `website` repo, but the Grafana one isn't. Not sure if that changes your thoughts here.

Reply:
We do note that Grafana Cloud documentation is maintained in the website repository in https://grafana.com/docs/writers-toolkit/contribute-documentation/#:~:text=For%20Grafana%20Cloud%20documentation%2C%20most%20content%20is%20managed%20in%20the%20website%20repository.%20Some%20content%20is%20sourced%20from%20other%20projects%2C%20listed%20below%3A.

It also mentions which parts come from OSS docs. I'll make a note to update that list after the reorganization is complete.

Reply:
TBH, I think the repo/project matching is a beyond the scope of this PR.